### PR TITLE
[EM-165] Update pull request size on every push

### PR DESCRIPTION
**Pull request by SandroDamilano**
## What does this PR do?
It catches all `push` events and creates a `Push` record for each one, associating it to its project, user and pull request (if any). If there's a pull request, its pull request size metric is updated 

Resolves [#165](https://rootstrap.atlassian.net/browse/EM-165)

### CODE REVIEW

File: db/migrate/20200908121903_add_branch_to_pull_requests.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddBranchToPullRequests < ActiveRecord::Migration[6.0]
+  def change
+    add_column :pull_requests, :branch, :text
```
Comment (horacio) [inline db/migrate/20200908121903_add_branch_to_pull_requests.rb:3]: Just curious, why `text` instead of `string`?

Comment (SandroDamilano) [reply db/migrate/20200908121903_add_branch_to_pull_requests.rb:3]: Nothing in particular. Would you recommend using string?

Comment (horacio) [reply db/migrate/20200908121903_add_branch_to_pull_requests.rb:3]: I think they're effectively pretty similar, so I'll use whatever it's being used for this kind of values ✨

Comment (SandroDamilano) [reply db/migrate/20200908121903_add_branch_to_pull_requests.rb:3]: They are both being used... although `string` seems to be more popular :P
I'll see if I can quickly change it

**Review by horacio — APPROVED** [review]

File: app/services/action_handlers/push.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+module ActionHandlers
+  class Push < ActionHandler
+    private
+
+    def handleable?
```
Comment (hdamico) [inline app/services/action_handlers/push.rb:5]: What is this for?

**Review by hdamico — APPROVED** [review]
Super clear!!

File: app/services/action_handlers/push.rb (hunk 2 above)
Comment (SandroDamilano) [reply app/services/action_handlers/push.rb:5]: The superclass, `ActionHandler` has the `call` method defined like this:
```
def call
    return unless handleable?

    handle_action
end
```
The rest of the subclasses use it to check if the action is handleable, but for push events there's only one possible event, which is already being handled. So for them, this method should always return `true`

**Pull request merged**